module/VuFind/src/VuFind/I18n/Translator/TranslatorAwareTrait.php

```diff
@@ -101,6 +101,10 @@ public function getTranslatorLocale($default = 'en')
      */
     public function translate($target, $tokens = [], $default = null)
     {
+        if ($this->translator !== null && $this->translator->getLocale() == "keys") {
+            return "*$target*";
```

Comment:
Would it make sense to embed the $tokens in the return value if the array is non-empty? Otherwise, key parts of many strings will not be displayed at all.

Reply:
Good call. Now, the tokens are also displayed.